.github/labeler.yml

```diff
@@ -0,0 +1,19 @@
+area/build:
+- build/**/*
+area/ci:
+- .travis
+- .github/**/*
+area/components:
+- drivers/storage/portworx/component/**/*
```

Comment:
In future we could create area for different components like CSI, Security, etc. I think it's okay for now.